packages/indexer-agent/src/network.ts

```diff
@@ -391,11 +396,12 @@ export class Network {
       const result = await this.subgraph
         .query(
           gql`
-            query allocations($indexer: String!, $disputableEpoch: Int!) {
+            query allocations($indexer: String!, $disputableEpoch: Int!, $minimumQueryFeesCollected: BigInt!) {
               allocations(
                 where: {
                   indexer: $indexer
                   closedAtEpoch_lte: $disputableEpoch
+                  queryFeesCollected_gt: $minimumQueryFeesCollected
```

Comment:
I think `gte` may be more intuitive here. If you set the threshold to e.g. 1 GRT but allocations with 1 GRT of fees are not collected, you might end up confused.